src/kmer.jl

```diff
@@ -473,6 +473,21 @@ end
     return (head, tail...)
 end
 
+
+"""
+    Base.rand(::Type{Kmer{A,K,N}}) where {A,K,N}
+    Base.rand(::Type{Kmer{A,K}}) where {A,K}
+
+Create a random kmer of a specified alphabet and length
+
+# Examples
+```jldoctest
+julia> import Random; Random.seed!(1); rand(Kmer{DNAAlphabet{2}, 3})
```

Comment:
I think rather move the seed stuff to a setup block ([see here](https://documenter.juliadocs.org/stable/man/doctests/#Setup-Code)).

Also, random seeds are not guaranteed stable across julia versions, so this is likely to break. Might be worth pulling in[ StableRNGs.jl](https://github.com/JuliaRandom/StableRNGs.jl).